### PR TITLE
Fixed issue #83. Added links

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes Issue #83 
Added social profile links
Links added:
https://trello.com/vladokoljibabic
https://yacy.net/profile/MChristen-Profile.pdf
https://vimeo.com/yacy
https://www.crunchbase.com/person/michael-meskes
https://mozillians.org/u/dietrich
https://autonome.wordpress.com/
https://opensourcebridge.org/users/1175
https://www.facebook.com/andrew.selle
https://github.com/ppaeps
https://wiki.freebsd.org/PhilipPaeps

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
preview link ?

**Comment by arjunpandey2003** [comment]
https://arjunpandey2003.github.io/2017.fossasiap.org/.

**Comment by arjunpandey2003** [comment]
@YashKumarVerma and @harshit98 please review with changes

**Review by harshit98 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -606,6 +607,9 @@ <h1>Featured Speakers</h1>
 											<a target="default" href="https://twitter.com/0rb1t3r"><i class="icon social_twitter"></i></a>
 											<a target="default" href="https://www.linkedin.com/in/orbiter/"><i class="icon social_linkedin"></i></a>
 											<a target="default" href="https://www.facebook.com/michael.christen"><i class="icon social_facebook"></i></a>
+											<a href="https://yacy.net/profile/MChristen-Profile.pdf"><i class="icon fa fa-external-link"></i></a>
+											<a href="https://vimeo.com/yacy"><i class="icon fa fa-external-link"></i></a>
+
```
Comment (amryarios) [inline index.html:612]: Why did you add this line?

**Review by amryarios — COMMENTED** [review]
why
